src/content/en/2020/cms.md

```diff
@@ -8,18 +7,378 @@ authors: [alexdenning]
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=893606466&format=interactive",
+  sheets_gid="465267881",
+  sql_file="core_web_vitals_distribution.sql"
+  )
+}}
+
+The FID scores are generally good here, but this does not translate to similar LCP scores. As suggested, the weight of individual pages on CMSs in addition to mobile connection quality, could play a role in the performance gaps that we see here. There is a small margin of difference between the resources shipped to desktop and mobile versions of a website. Last year we noted that optimizing for the mobile experience was necessary. Average scores have increased on desktop and mobile, but further attention is required here.
+
+**Cumulative Layout Shift**
+
+Cumulative Layout Shift (CLS) measures the instability of content on a web page within the first 500ms of user input. CLS measures any layout changes which happen after user input. This is important on mobile in particular, where the user will tap where they want to take an action – such as a search bar – only for the location to move as additional images, ads, or similar load.
+
+A score of 0.1 or below is good, over 0.25 is poor, and anything in between needs improvement.
+
+{{ figure_markup(
+  image="cms-real-user-cumulative-layout-shift-experiences.png",
+  caption="Real-user Cumulative Layout Shift experiences.",
+  description='Bar chart showing the top 5 CMSs and whether they have a "good" Cumulative Layout Shift experience. Wordpress has a 47% of desktop sites with a "good experience" and 57% of mobile sites. Drupal has 58% for desktop and 70% for mobile, Joomla has 51% for desktop and 63% for mobile, Squarespace has 35% for desktop and 44% for mobile, and Wix has 58% for desktop and 59% for mobile.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=1265001868&format=interactive",
+  sheets_gid="465267881",
+  sql_file="core_web_vitals_distribution.sql"
+  )
+}}
+
+The top 5 CMSs do poorly here. Only 50% of web pages loaded by a top 5 CMS have a “good” CLS experience, with this figure rising to 59% on mobile. Across all CMSs the average desktop score is 59% and average mobile score is 67%. This shows us all CMSs have work to do here, but the top 5 CMSs in particular need improvement. 
+
+**Lighthouse scores**
+
+[Lighthouse](https://almanac.httparchive.org/en/2019/methodology#lighthouse) is an open-source, automated tool designed to help developers assess and improve the quality of their websites. One key aspect of the tool is that it provides a set of audits to assess the status of a website in terms of performance, accessibility, progressive web apps, and more. For the purposes of this chapter, we are interested in two specific audits categories: PWA and accessibility.
```

Comment:
Accessibility I can understand but why are you interested in PWA?

Lighthouse itself doesn't show the PWA score and it's a bit of a niche technology (as shown by the PWA chapter). Is there an expectation that if you use a CMS you get good PWA capabilities?

Would Accessibility and SEO be better ones to concentrate on? Performance is also important but that's covered elsewhere.

Reply:
Especially given that there's not much difference between the CMS's for PWA so is there really much to talk about here since it's a niche technology, the CMSs are all roughly the same, and I'm not sure there is an expectation that using a CMS will automatically give you PWA capabilities.

I've run the SEO stats and that gives this chart:

![SEO stats by CMS](https://user-images.githubusercontent.com/10931297/99654481-49df7c00-2a52-11eb-936c-5bf6e98f411a.png)

So a bit more variation so maybe more interesting to talk about than PWA as I think there **IS** an expectation that a CMS should give you good SEO capabilities.

Or alternatively we include both?

Reply:
I think I included this because it was on the list from last year. This isn't my area of expertise, so I leaned heavily on what happened last time. I don't really feel qualified to make judgements or changes here to be honest, so happy to proceed as is thought best.

Reply:
Ah I see. They did use different stats last year which were a bit more interesting than the ones we used this year so ours are a bit less interesting on this topic.

I still don't think a CMS automatically implies PWA capabilities so still don't think it really belongs in this chapter. Especially now we have less interesting stats to report on.

@ernee / @amedina as the 2019 authors do you have any more insight what you were trying to show with this?

I think SEO is more meaningful, and then also gives us a nice change as something new from last year. I added this in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1513/commits/94cfbc95f43a1e4d2b94888a368e9b87107404f4 if you could have a read @alexdenning to make sure you're comfortable with the words I'm putting in your mouth here! 😀